Userland/Libraries/LibWeb/Layout/BlockFormattingContext.cpp

```diff
@@ -137,10 +137,16 @@ void BlockFormattingContext::compute_width(Box const& box, AvailableSpace const&
         // FIXME: This const_cast is gross.
         const_cast<ReplacedBox&>(replaced).prepare_for_replaced_layout();
         compute_width_for_block_level_replaced_element_in_normal_flow(replaced, remaining_available_space);
-        // NOTE: We don't return here.
+        if (box.is_floating()) {
+            // 10.3.6 Floating, replaced elements:
+            // https://www.w3.org/Style/css2-updates/css2/visudet.html#float-replaced-width
```

Comment:
could you use link to not editor's draft but relevant spec https://www.w3.org/TR/CSS22/

Reply:
also you could reference relevant text from spec here so it's understandable why early return makes sense here.

Reply:
> could you use link to not editor's draft but relevant spec https://www.w3.org/TR/CSS22/

Done.

Reply:
> also you could reference relevant text from spec here so it's understandable why early return makes sense here.

Not sure how to summarize it, they're simply different algorithms. The early return was present before this change as well, for both floating cases. I'm merely separating the replace and non-replaced algorithms for the floating case, per the specifications.